pages/index.tsx

```diff
@@ -1,12 +1,22 @@
 import type { NextPage } from "next";
 import Link from "next/link";
-import { Container } from "react-bootstrap";
+import {Container, Image} from "react-bootstrap";
+import React from "react";
 
 const Home: NextPage = () => {
   return (
     <Container>
-      <h1>Hello, world!</h1>
-      <Link href="/about">About</Link>
+      <Image
+          src="/blueno.png"
+          alt="Blueno logo"
+          width="300"
+          height="300"
+          className="d-inline-block align-top"
+      />
+      <h1>Wordify</h1>
+        <a href="/game" className="btn btn-secondary btn-lg active" role="button" aria-pressed="true">Play As Guest</a>
```

Comment:
You should consider using the Next.js `<Link>` component rather than an `<a>` tag - the `<Link>` component will take advantage of the Next.js router.